### ISSUE TITLE
The default value of delegatesFocus is not very reasonable

### ISSUE THREAD

**Issue by ShineKidd** [alex-vukov/css-boundary#1]
https://github.com/alex-vukov/css-boundary/blob/5e0109e72bef183f0657d69360df8b0d25b9f76c/src/styleLoader.ts#L12

The option `delegatesFocus` is set to `true` that the user cannot select text.  

check the follow example https://codepen.io/liamdebeasi/pen/xxYMvLB

Maybe we can make users customize this option.

```js
createShadowInstance(parentElementId: string, options: { delegatesFocus: boolean  })
```

**Comment by alex-vukov** [comment]
Thank you for your feedback and for the example! I am able to select the text but I agree that `delegatesFocus` should be configurable and should not have `true` as a default value. I will prepare a new version with this change and I will close the issue when it's released.

**Comment by xianyusPadding** [comment]
Is there any progress?

**Comment by alex-vukov** [comment]
Yes! I just pushed a fix in version [1.0.2](https://www.npmjs.com/package/css-boundary) and it is released. The default value of delegatesFocus will be `false`. You can also now pass an optional init object to override this if you need to. I also updated the Readme with this information!
